src/js/main.js

```diff
@@ -317,6 +317,12 @@ if (document.body.dataset.page === "integrations") {
     }
   });
 
+  const params = new URLSearchParams(window.location.search);
+  const integration = params.get("integration");
+  if (integration) {
+    filterSelection(integration);
+  }
+
```

Comment:
We can also make it such that clicking on integration type updates the URL automatically. In `filterSelection` you can do:

```js
const url = new URL(window.location.href);

if (integrationType === "all") {
  url.searchParams.delete("integration");
} else {
  url.searchParams.set("integration", integrationType);
}

history.pushState(null, '', url.toString());
```